doc/UserGuide.md

```diff
@@ -68,6 +74,7 @@ Examples:
 
 ### Deleting a person : `delete`
 Deletes the specified person from the address book. Irreversible.<br>
+After successful deletion of the person, the current updated address book will be listed.<br>
 Format: `delete INDEX`
```

Comment:
Good job updating the user guide :) 

Just to nitpick, you can try to keep the documentation style consistent (particularly for the `sort` command -- a full stop, a spacing etc.)